com.microsoft.mrtk.graphicstools.unity/Runtime/Experimental/Magnifier/Scripts/MagnifierManager.cs

```diff
@@ -172,7 +178,27 @@ public void ApplyMagnification()
         {
             Shader.SetGlobalFloat(MagnificationPropertyName, 1.0f - Magnification);
         }
-
+        public void GetMousePos()
+        {
+            if (inLensMode)
+            {
+                Vector3 wldpos = Camera.main.ScreenToViewportPoint(Input.mousePosition);
```

Comment:
nit: change the variable name to `viewportPosition` since it's not a world position. 
```suggestion
                Vector3 viewportPosition = Camera.main.ScreenToViewportPoint(Input.mousePosition);
```

Reply:
I'm a bit worried that `Input.mousePosition` will only be valid for projects using Unity's old input system. Could you get the mount position like [this](https://github.com/microsoft/MixedReality-GraphicsTools-Unity/blob/6c6f65d70e2a2f87bd1cbeb2a94aeffd8254e9d8/com.microsoft.mrtk.graphicstools.unity/Samples~/UnityUI/Scripts/CursorController.cs#L73)?

```
            Vector3 cursorPosition;
#if USE_INPUT_SYSTEM
            Vector2 position2D = Mouse.current != null ? Mouse.current.position.ReadValue() : Vector2.zero;
            cursorPosition = new Vector3(position2D.x, position2D.y, 0.0f);
#else
            cursorPosition = Input.mousePosition;
#endif // USE_INPUT_SYSTEM
```